emails/payin_bankwire_succeeded.spt

```diff
@@ -1,9 +1,8 @@
-{{ _("The bank wire to your Liberapay wallet has succeeded") }}
+{{ _("The bank wire to your Liberapay wallet cleared") }}
 
 [---] text/html
 <p>{{ _(
-    "We're happy to report that the transfer from your bank account has succeeded. "
-    "Of the {0} that were sent, {1} went into your Liberapay wallet, and {2} were paid in fees."
+    "Of the {0} sent, {1} went into your Liberapay wallet, and {2} was paid in fees."
```

Comment:
We already know the bank wire cleared. It is implied it came from a bank account, and that it was yours. It could be someone elses bank account too, but if the user gets this far, that much is understood already. So this whole first segment is redundant and irrelevant.

"that were" is redundant, and it should be "was" at the end because it is "it" money